src/pages/exception/500/locales/en-US.ts

```diff
@@ -0,0 +1,4 @@
+export default {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=4583897)